imagersite/imager_profile/templates/profile.html

```diff
@@ -0,0 +1,14 @@
+{% extends "../../imagersite/base.html" %}
```

Comment:
You don't need the relative path to the html template. You could just have it as `imagersite/base.html`